src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populations = [...document.querySelectorAll('.population')];
+
+const totalPopulation = populations.reduce((prev, population) => {
+  const populationToNumber = Number(population.textContent.replace(/,/g, ''));
+
+  return populationToNumber + prev;
+}, 0);
+
+const averagePopulation = totalPopulation / populations.length;
```

Comment:
There is a potential issue here if the 'populations' array is empty. Dividing by zero will result in 'Infinity'. Consider adding a check to ensure 'populations.length' is not zero before performing the division.